README.md

```diff
@@ -41,6 +41,21 @@ See [`allTargets`](https://github.com/lgeiger/node-abi#usage) for all available
 
 For more options run `prebuild --help`. The prebuilds created are compatible with [node-pre-gyp](https://github.com/mapbox/node-pre-gyp)
 
+If you'd like to include other files with your prebuilds like additional
+`.node` files or other native libraries, you can pass a file-matching regular
+expression to the  `--collect-files-filter` parameter:
+
+```
+prebuild -t 8.0.0 --collect-files-filter "\.(node|a)$"
+```
+
+Note that if you include multiple `.node` files, you will need to use the
+prebuild-install's '`--binary-name` parameter to indicate which file should be
```

Comment:
Looks like there's a `'` hanging loose there, just before `--binary-name` :)

Reply:
Whoops, thanks!  Also, what do you think about `--binary-name` as the `prebuild-install` parameter for choosing the right .node file?  Is there a more obvious name?